SIDFactoryII/source/utils/global.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+namespace Foundation
+{
+	class IPlatform;
+}
+
+namespace Utility
+{
+	class ConfigFile;
+}
+
+namespace Utility
+{
+	// Singleton for global configuration
+	class Global
+	{
+
+	public:
+		static Global& instance();
+		Global(Global& inOther) = delete;
+		Global(Global&& inOther) = delete;
+		Global(const Global& inOther) = delete;
```

Comment:
Same as the other constructors with the same pattern.

Reply:
Deleted https://en.cppreference.com/w/cpp/language/move_assignment operator
Removed the line `Global(const Global& inOther) = delete;`